Dockerfile

```diff
@@ -1,2 +1,4 @@
 FROM yastdevel/ruby
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
```

Comment:
The `--gpg-auto-import-keys` option is not needed anymore. The GPG key is already imported, this is obsolete. (In the past the key was imported by zypper, I have improved that.)